tests/linux-sandbox-cert-test.nix

```diff
@@ -1,29 +1,30 @@
-{ fixed-output }:
+{ mode }:
 
 with import ./config.nix;
 
-mkDerivation ({
-  name = "ssl-export";
-  buildCommand = ''
-    # Add some indirection, otherwise grepping into the debug output finds the string.
-    report () { echo CERT_$1_IN_SANDBOX; }
+mkDerivation (
```

Comment:
Nit, but I actually like the old indent style better :D

Reply:
Me too, but I will stick with nixpkgs-fmt for now.